osu.Game/Screens/Edit/Timing/TapButton.cs

```diff
@@ -295,6 +295,9 @@ public void OnReleased(KeyBindingReleaseEvent<GlobalAction> e)
 
         private void handleTap()
         {
+            if (selectedGroup?.Value == null)
+                return;
```

Comment:
This one looks redundant, as the only other usage of `selectedGroup` in this method is null-propagated and checked anyway.

https://github.com/ppy/osu/blob/8d0fe61aff00014dae8ef4fe1b0ad3fb0d896abf/osu.Game/Screens/Edit/Timing/TapButton.cs#L316-L322

Reply:
I saw that, but it's only checking the DI level nullability (I guess for tests?), not the `Value` portion so saw value in adding this check.